templates/ec2-bootstrap-trigger.yaml

```diff
@@ -32,13 +28,10 @@ Resources:
       InstanceType: t3.medium
       ImageId: ami-03ededff12e34e59e
       IamInstanceProfile: !Ref IAMInstanceProfile
-      KeyName: !Ref PrivateKey
-      SecurityGroupIds:
-        - !Ref SecurityGroup
+      SubnetId: !Ref SubnetId
```

Comment:
Follow example here: https://github.com/nlpsandbox/nlpsandbox-infra/blob/2155e67a552cb7cdaf9237e0bf889ac3f901ea3c/sceptre/nlpsandbox/templates/nlpsandbox.yaml#L121-L127

Reply:
@philerooski the only thing left is to not only assign this instance a private subnet but to follow the example above to assign it a security group and VPC.

Reply:
The subnet belongs to the VPC, so putting the EC2 in the subnet puts it in the VPC as well. I see there are multiple types of security groups associated with the VPC. The instance ends up in the default security group for the VPC, but I'm not sure which is the right security group to use.
<img width="552" alt="image" src="https://user-images.githubusercontent.com/2773709/167519732-7c360330-bb01-4755-95f4-fc74d9b5d081.png">

Reply:
You'll want the VpnSecurityGroup.